src/content/docs/en/technology/overview/scroll-upgrades.mdx

```diff
@@ -19,6 +21,69 @@ The following contracts are used to initiate upgrades and execute upgrades after
 
 You can join our [Telegram channel for technical updates](https://t.me/scroll_tech_updates), which includes future upgrade announcements and on-chain operation events.
 
+## Euclid Upgrade
+
+<Aside type="note" title="Note">
+  Detailed impacts and action items for partner can be found in this [Note for Partners](https://scrollzkp.notion.site/Euclid-Upgrade-5-Note-for-Partners-1ad7792d22af808cad5ac643a5ecdaf0)
```

Comment:
Can you copy the content from "Note for Partners" to here. That's what we do in the past.

For each upgrade, we usually have 4 sections:
1. overview
2. timeline
3. technical details
4. compatibilities

you can find item 1/2/4 from note to partners. For 3 I don't think we provide this time so you can skip it.